test/integration/targets/test_infra/runme.sh

```diff
@@ -21,3 +21,7 @@ echo "rc was $APB_RC (must be non-zero)"
 echo "ensure playbook output shows assert/fail works (True)"
 echo "$PB_OUT" | grep -F "fail works (True)" || exit 1
 echo "$PB_OUT" | grep -F "assert works (True)" || exit 1
+
+# ensure test-module script works well
+PING_MODULE_PATH="$(pwd)/../../../../lib/ansible/modules/system/ping.py"
+../../../../hacking/test-module -m "$PING_MODULE_PATH" -I ansible_python_interpreter=$(which python)
```

Comment:
> test/integration/targets/test_infra/runme.sh:27:86: SC2046 Quote this to prevent word splitting.

@HD650 this linter message means that in the event of command in `$(...)` expression returning smth with space the command fill likely break. For example, if it returns `/usr/bin/env python` it will result in smth like `...  -I ansible_python_interpreter=/usr/bin/env python`, which is not what you want. Instead, you expect all of it to go to the same arg, like `...  -I ansible_python_interpreter="/usr/bin/env python"`.

To fix this, just wrap argument with double quotes `"` (single quotes prevent bash expression interpolation, which will break things).